core/controllers/base.py

```diff
@@ -37,6 +37,7 @@
 import utils
 
 from google.appengine.api import users
+from google.appengine.ext.webapp import template
```

Comment:
Imports should always be at module level. Is "template" a module? If not, do "from google.appengine.ext import webapp" and make use of webapp.template.

Reply:
Yes, template is a module